### PR TITLE
Increase number of mac taskcluster WPT jobs

### PR DESCRIPTION
**Pull request by jdm**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/23355)

### CODE REVIEW

**Comment by jdm** [comment]
Since there are now three macs available and the unit testing one usually finishes before the release build, we should be able to run all three jobs in parallel when the release build job is complete.

File: etc/taskcluster/macos/config/roster [hunk 1]
```diff
@@ -2,3 +2,5 @@ mac1:
     host: servo-tc-mac1.servo.org
 mac2:
     host: servo-tc-mac2.servo.org
+mac3:
+    host: servo-tc-mac3.servo.org
```
Comment (Manishearth) [inline etc/taskcluster/macos/config/roster]: newline at eof?

**Review by Manishearth — APPROVED** [review]
r=me

**Comment by CYBAI** [comment]
@bors-servo retry

```
  ▶ FAIL [expected PASS] /css/CSS2/positioning/position-relative-031.xht
  └   → /css/CSS2/positioning/position-relative-031.xht 36dcbd59a9c328d2413156c5680ea54954206622
/css/CSS2/positioning/position-relative-031-ref.xht 8b4568e7196673fa5ee61a408582bf8ed45fc90e

```

**Comment by jdm** [comment]
@bors-servo r-
Gotta look into the GStreamer panics.

**Comment by jdm** [comment]
Problems with:
* /html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-*

**Pull request merged**